scripts/boot-master/copy_cluster_skel.sh

```diff
@@ -25,4 +25,4 @@ ensure_directory_reachable ${cluster_dir}
 sudo mv /tmp/icp/cluster/* ${cluster_dir}
 
 # Take a backup of original config file, to keep a record of original settings and comments
-cp ${cluster_dir}/config.yaml ${cluster_dir}/config.yaml-original
+sudo cp ${cluster_dir}/config.yaml ${cluster_dir}/config.yaml-original
```

Comment:
Is this command necessary to use? I thought that the directory should be read/writeable and accessible by `$(whoami)` because of the lines above

```
sudo chown $(whoami):$(whoami) -R /tmp/icp
ensure_directory_reachable ${cluster_dir}
sudo mv /tmp/icp/cluster/* ${cluster_dir}
```

If these lines did not do what I thought they actually did, then we should resolve that. 😄

Reply:
@arewm you are right. I was referring to the code in 3.0.8 version where chown is before the cluster folder is fetched from docker image. Closing this PR.